src/hoc/ThemeProvider/ThemeProvider.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file ThemeProvider.js
+ * Exports a HOC ThemeProvider component that attaches children their MUI theme.
+ */
+
+import React from 'react';
+import { MuiThemeProvider } from '@material-ui/core';
+
+import muiTheme from '../../styles/muiTheme';
+
+const ThemeProvider = WrappedComponent => props => (
```

Comment:
ThemeProvider hoc adds MuiThemeProvider wrapper (with the global theme) to it's wrapped component. This is especially useful in tests for components that require the MUI theme.